docker-compose.yml

```diff
@@ -15,5 +16,15 @@ services:
     volumes:
       - postgres_data:/var/lib/postgresql/data
 
+  signal:
+    image: bbernhard/signal-cli-rest-api:latest
+    environment:
+      - USE_NATIVE=0
+        #- AUTO_RECEIVE_SCHEDULE=0 22 * * * #enable this parameter on demand (see description below)
```

Comment:
Yes, because I doubt this feature can process the data in the way we want it? But I would be glad about anyone who looks into this feature and sees if it has any value for us.

Reply:
I checked this some time ago. The only use case is if you do not need to process incoming messages. (Signal requires that you regularly connect to the servers to receive messages even if you're not interested in them/don't receive any.)